ch3/upload_model_fluent.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+import mlflow
+import pandas as pd
+import psycopg2
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import SVC
+
+# 0. set mlflow environments
+MLFLOW_TRACKING_URI = "http://localhost:5000"
+MLFLOW_S3_ENDPOINT_URL = "http://localhost:9000"
+
+os.environ["MLFLOW_S3_ENDPOINT_URL"] = MLFLOW_S3_ENDPOINT_URL
+os.environ["MLFLOW_TRACKING_URI"] = MLFLOW_TRACKING_URI
+os.environ["AWS_ACCESS_KEY_ID"] = "minio"
+os.environ["AWS_SECRET_ACCESS_KEY"] = "miniostorage"
+
+# 1. get data
+db_connect = psycopg2.connect(
+    user="myuser",
+    password="mypassword",
+    host="localhost",
+    port=5432,
+    database="mydatabase",
+)
+df = pd.read_sql("SELECT * FROM iris_data ORDER BY id DESC LIMIT 100", db_connect)
+X = df.drop(["id", "target"], axis="columns")
```

Comment:
```suggestion
df = pd.read_sql("SELECT * FROM iris_data ORDER BY id DESC LIMIT 100", db_connect)

X = df.drop(["id", "target"], axis="columns")
```
여기서 띄어쓰기 한번 하면 좋을 것 같아요~